File: backend/src/main/java/com/revnu/backend/features/sales/dto/SaleResponse.java
package com.revnu.backend.features.sales.dto;

import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.UUID;

import com.revnu.backend.features.sales.model.SaleStatus;

public record SaleResponse(
        UUID id,
        BigDecimal amount,
        UUID categoryId,
        String categoryName,
        String notes,
        SaleStatus status,
        LocalDateTime createdAt
        ) {

}

File: mobile/app/src/main/java/com/revnu/mobile/features/sales/model/SaleResponse.kt
package com.revnu.mobile.features.sales.model

import com.google.gson.annotations.SerializedName

data class SaleResponse(
    @SerializedName("id") val id: String,
    @SerializedName("amount") val amount: Double,
    @SerializedName("categoryId") val categoryId: String,
    @SerializedName("categoryName") val categoryName: String?,
    @SerializedName("notes") val notes: String?,
    @SerializedName("status") val status: String,
    @SerializedName("createdAt") val createdAt: String
)